### PR TITLE
Verbosity was missing; POSIX operations

### PR DESCRIPTION
**Pull request by polluks**
l &rarr; t
a &rarr; r

### CODE REVIEW

**Comment by greg-king5** [comment]
Did you read the wiki link that Oliver posted in PR #442?  It describes how you could have used the Github web interface to add this file's patches to PR #443.

File: doc/ar65.sgml [hunk 1]
```diff
@@ -32,16 +32,17 @@ for the cc65 compiler. ar65 is part of this suite.
...
+		r	Add modules
+		d	Delete modules
+		t	List library table
+		v	Increase verbosity (put before other operation)
+		x	Extract modules
+		V	Print the archiver version
 </verb></tscreen>
 
-You may add modules to a library using the `a' command. If the library
+You may add modules to a library using the `r' command (`a' is deprecated). If the library
```
Comment (spiro-trikaliotis) [inline doc/ar65.sgml]: I would expect the 'a' to be mentioned in the list of operations, too, with a remark that it is deprecated.

File: doc/ar65.sgml [hunk 2]
```diff
@@ -81,13 +82,13 @@ This will delete the module named `sub1.o' from the library, printing an
 error if the library does not contain that module.
 
 
-The `l' command prints a list of all modules in the library. Any module
-names on the command line are ignored.
+The `t' command prints a table of all modules in the library (`l' is deprecated).
```
Comment (spiro-trikaliotis) [inline doc/ar65.sgml]: Same here: I would expect the 'l' to be mentioned in the list of operations, too, with a remark that it is deprecated.

Comment (polluks) [inline doc/ar65.sgml]: I know what you mean.
Let me tell you about the Wikipedia article of X-Window. A the beginning was "X-Window**s** spelling is wrong". If something is wrong or deprecated, don't put it on the top.

**Pull request merged**